sdv/metadata/metadata.py

```diff
@@ -70,6 +70,10 @@ def _set_metadata_dict(self, metadata, single_table_name=None):
         else:
             if single_table_name is None:
                 single_table_name = self.DEFAULT_SINGLE_TABLE_NAME
+                warnings.warn(
+                    'Did not assign a table name to single table. '
```

Comment:
maybe we can be slightly more verbose
```suggestion
                    'No table name was provided to metadata containing only one table.  '
```